deepchem/utils/pytorch_utils.py

```diff
@@ -14,3 +14,60 @@ def get_activation(fn: Union[Callable, str]):
     if isinstance(fn, str):
         return getattr(torch.nn.functional, fn)
     return fn
+
+
+def unsorted_segment_sum(data, segment_ids, num_segments):
+    """Computes the sum along segments of a tensor. Analogous to tf.unsorted_segment_sum.
+
+    Parameters
+    ----------
+    data: A tensor whose segments are to be summed.
```

Comment:
The formatting here is off. You should have:

```
data: torch.Tensor
  A tensor whose ....
```

with the type annotated. Check https://numpydoc.readthedocs.io/en/latest/format.html for more examples